### DISCUSSION TITLE
add a command

### DISCUSSION THREAD

**Discussion by iamdogshi** [winstxnhdw/lc-hax#499] [General]
i was wondering if you could add a command which could make easter eggs always explode on drop or throw and for it to be used more than once

**Comment by winstxnhdw** [comment]
This is v50, yes?

**Reply by iamdogshi** [reply]
yes

**Reply by iamdogshi** [reply]
seeing as v50 is out, would this be possible?

**Reply by winstxnhdw** [reply]
Not sure. I’ll take a look.

**Reply by xAstroBoy** [reply]
easter eggs are holding the Stun Grenade explosion component.

**Reply by xAstroBoy** [reply]
No, you cannot make eggs explode on command unless host only
